ephios/static/ephios/js/formset/formset.js

```diff
@@ -0,0 +1,379 @@
+/**
```

Comment:
as far as I see you didn't remove the packaged version from the dependencies

Reply:
correct, and we actually still depend on it for the template tags. Trust me I tried to remove it ;)

Reply:
sad story. did you commit your changes upstream?

Reply:
not yet :sweat_smile:

Reply:
okay but wth is this under ephios and not seperate like every other dependency?